pkg/controller/storagecluster/cephcluster.go

```diff
@@ -564,3 +575,43 @@ func (r *ReconcileStorageCluster) checkTuneStorageDevices(ds ocsv1.StorageDevice
 	// not a known disk type, don't tune
 	return diskSpeedUnknown, nil
 }
+
+func generateStretchClusterSpec(sc *ocsv1.StorageCluster) *cephv1.StretchClusterSpec {
```

Comment:
```suggestion
func generateStretchClusterSpec(sc *ocsv1.StorageCluster) cephv1.StretchClusterSpec {
```

Reply:
cephv1.MonSpec requires *cephv1.StretchClusterSpec. It just makes it easy to match with what Cephv1 defines it as.

Reply:
I think this is fine in this case.